keras_vggface/tensorflow_lite_conversion.py

```diff
@@ -0,0 +1,45 @@
+from keras_vggface import VGGFace
+import tensorflow as tf
+from models import preprocessing
+
+
+def create_tflite_model_file(keras_model, filename):
+  """Converts keras model into TensorFlow lite model, and
```

Comment:
📝 🧐 This looks like it uses 2 space rather than 4 space indentation, can you reformat this file?

Reply:
Yup, i fixed it now 🍌